CONTRIBUTING.rst

```diff
@@ -256,8 +263,54 @@ When developing, you can make Python changes available in R by running (from mlf
 Please also follow the recommendations from the
 `Advanced R - Style Guide <http://adv-r.had.co.nz/Style.html>`_ regarding naming and styling.
 
+Java
+~~~~
+
+If contributing to MLflow's Java APIs or modifying Java documentation,
+install `Java <https://www.java.com/>`_ and `Apache Maven <https://maven.apache.org/download.cgi>`_.
+
+Certain MLflow modules are implemented in Java, under the ``mlflow/java/`` directory.
+These are the Java Tracking API client (``mlflow/java/client``) and the Model Scoring Server
+for Java-based models like MLeap (``mlflow/java/scoring``).
+
+Other Java functionality (like artifact storage) depends on the Python package, so first install
```

Comment:
Can we replace this sentence ("Other Java functionality (like artifact storage) depends on the Python package, so first install the Python package in a conda environment as described above.") with a link to the "Common prerequisites and dependencies" section, as you've done for Python [here](https://github.com/mlflow/mlflow/pull/3219/files#diff-ea1f65a8eaac0cec7ae151dfdbbbc587R292)? And could we do the same for R?

Reply:
sounds good; will do it.